.github/workflows/compareNDandODbuilds.yml

```diff
@@ -2,24 +2,108 @@ name: Build and profile Android apps
 
 on:
   workflow_dispatch:
+    inputs:
+      PULL_REQUEST_NUMBER:
+        description: Pull Request number for correct placement of app
```

Comment:
What does "placement of app" mean?

Reply:
It means that we want to build app from given commit

Reply:
OK, what about changing this description this?

> Pull request number to build the app from

Reply:
Good idea, thanks!